src/ui/profile_preferences.ui

```diff
@@ -1413,11 +1413,14 @@
          <layout class="QGridLayout" name="gridLayout_22">
           <item row="0" column="0">
            <widget class="QCheckBox" name="mMapperUseAntiAlias">
+            <property name="toolTip">
+             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This enables anti-aliasing (AA) for tha map view, making it look smoother and nicer. Disable this if you're on a very slow computer.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Is this for just on-map text or for all graphics and is it just for the 2D mapper or both - we might need to clarify/check?  Also, there is usually a cost for doing Anti-Aliasing so we might want to admit that _slooooww_ machines _might_ consider turning it off even if things look better with it.  This may be of significant for large to humongous maps where the drawing time is significant.

_Note: I believe we (used to) boast that we cached the mapped display as a pixmap and just manipulated that where possible to speed up screen update/redraw time - sadly we dropped that some time ago though I think it still features in our product "literature" IIRC - so this point might be of importance..._